setup.py

```diff
@@ -29,6 +29,7 @@
 Jupyter Notebooks to share resources across distributed clusters such as
 Apache Spark, Kubernetes and others..
 """,
+    long_description_content_type="text/plain",
```

Comment:
```suggestion
    long_description_content_type="text/markdown",
```

Reply:
`setup.py` currently uses [an _in-line_ long description](https://github.com/jupyter-server/enterprise_gateway/blob/00d7376b932eacd347b3c32c863691bfbad53b86/setup.py#L27-L30), so I used `text/plain` as the content type.  Seems like most PyPi projects point at their `README.md` file.  Is that what you had thought?  I'm good with using it as our long description - it winds up looking really clean - like server's looks: https://pypi.org/project/jupyter-server/

Reply:
Ah, fair, thanks~

Reply:
I think it is fine as-is, as long as it points back to the repo on PyPI

Reply:
> as long as it points back to the repo on PyPI

Confirmed.